src/components/collection/CollectionsGrid.tsx

```diff
@@ -40,8 +40,8 @@ const CollectionsGrid = ({
         justifyContent="flex-start"
         id={id}
       >
-        {Array.from({ length: 4 }).map(() => (
-          <Grid xs={6} sm={4} md={3} lg={3} xl={2}>
+        {Array.from({ length: 4 }, (_, idx) => idx).map((idx) => (
```

Comment:
I think in cases like this, we should pre-generate a list with fixed ids and use that to map over. Using the index completely defies the purpose of using a key in react.

Reply:
It's kinda the same, except I used index to fill the map.